src/components/ActivityMap/AltitudeMap.tsx

```diff
@@ -22,6 +23,29 @@ const BRAND_ORANGE = '#FF4500';
 // this controls how many points are skipped before rendering another
 const SAMPLING_RATE = 5;
 
+function matchMeanElevation(data: AltitudePoint[]): AltitudePoint[] {
+  const stravaMean = mean(
+    data.flatMap(({ stravaAltitude }) =>
+      !!stravaAltitude ? stravaAltitude : []
```

Comment:
Can this just be `!!stravaAltitude ?? []`

Reply:
Also why is the null case an empty array which is then flat mapped? Why not just leave it null or something and filter nulls?

Reply:
I think this is a holdover from a previous implementation where I was trying to get the sum with Array.reduce()
Will simplify implementation in a moment

Reply:
Going to leave this as-is for now. Flat map allows TS to correctly infer the type of the resulting array without type guards, which is (maybe?) more elegant than a .map -> .filter with type guards 🤷